lib/cisco_node_utils/cmd_ref/tacacs_global.yaml

```diff
@@ -0,0 +1,31 @@
+# enable_global.yaml
+---
+encryption:
+  set_value: "<state> tacacs-server key <option> <key>"
+
+key:
+  get_value: '/^tacacs-server key (\d+)\s+(\S+)/'
+  default_value: ""
+  nexus:
+    get_command: "show run tacacs all"
+  ios_xr:
+    get_command: "show running-config tacacs-server"
+
+key_format:
+  auto_default: false
+  get_value: '/^tacacs-server key (\d+)\s+(\S+)/'
+  default_value: 0
```

Comment:
You're using both `auto_default` and `default_value` ? This should be a one-or-the-other.